03/app.js

```diff
@@ -7,12 +7,44 @@ const c = randomNumber(min, max);
 
 console.log(a, b, c);
 
-
-
-
-
-
-
 function randomNumber(min, max) {
     return Math.round((Math.random() * (max - min)) + min);
-}
\ No newline at end of file
+}
+
+function getSum(a, b, c) {
+    if (a > b && a > c) {
+        return (b > c) ? (a + b) : (a + c);
+    } else if (b > c) {
+        return (a > c) ? (a + b) : (b + c);
+    } else {
+        return (a > b) ? (a + c) : (b + c);
+    }
+}
+const sum = getSum(a, b, c);
+
+
+const isEven = function (num) {
+    if (!(num * 1) && (+num !== 0)) {
+        return null;
+    } else {
+        return !(num % 2) ? "true" : "false"
+    }
+}
+
+const even = isEven(sum);
+
+function showInfo(num, par) {
+    switch (par) {
+        case 'false':
```

Comment:
Po zmianie wyżej tutja też trzeba wstawić `false`, a nie `'false'`. Ta druga opcja to po prostu normalny ciąg znaków co powoduje, że `'false'` zawsze daje prawdę.

Reply:
Zmienione